CMakeLists.txt

```diff
@@ -83,6 +102,34 @@ target_link_libraries(${PROJECT_NAME}_twist_merger
   ${UNIFIED_TELEOP_TARGET_LINK_LIBRARIES}
 )
 
+# lifecycle_point_stamped_incr
+add_executable(${PROJECT_NAME}_lifecycle_point_stamped_incr src/lifecycle_point_stamped_incr.cpp)
```

Comment:
executables should not be prefixed with ${PROJECT_NAME}. This is a holdover from ROS 1